.github/workflows/go.yml

```diff
@@ -0,0 +1,44 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, macOS-latest, windows-latest]
+
+    steps:
+    - name: Check out code into $GITHUB_WORKSPACE directory
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+
+    - name: Install Dependencies (Linux)
+      run: |
+        sudo apt-get install python-gnomekeyring
+        sudo apt-get install gnome-keyring
+        dbus-launch /usr/bin/python -c "import gnomekeyring;gnomekeyring.create_sync('login', '');"
+      if: matrix.os == 'ubuntu-16.04'
```

Comment:
Maybe add a comment with the link to why we must use 16.04 for now.

Reply:
done